server/app.coffee

```diff
@@ -1,2 +1,8 @@
 Meteor.startup ->
-    
+    config =
+        run_seed: process.env.SEED == "true"
+
+    if config.run_seed
+        Meteor.call 'seedPosts'
+        Meteor.call 'seedTopics'
+        Meteor.call 'seedSections'
```

Comment:
W sumie to na produkcji też powinien być seed. http://stackoverflow.com/questions/11183424/how-a-meteor-application-knows-if-its-running-on-development-test-or-productio